### ISSUE TITLE
Error on staging while installing bower deps

### ISSUE THREAD

**Issue by notthetup** [webuildsg/webuild#213]
There is an error as bower tries to create a directory inside a directory which it doesn't have access to.

```
remote: Error: EACCES: permission denied, mkdir '/var/lib/openshift/56df98da7628e17b0c0000a0/.config'
remote:     at Error (native)
remote:     at Object.fs.mkdirSync (fs.js:842:18)
remote:     at sync (/var/lib/openshift/56df98da7628e17b0c0000a0/app-root/runtime/repo/node_modules/bower/lib/node_modules/mkdirp/index.js:70:13)
remote:     at Function.sync (/var/lib/openshift/56df98da7628e17b0c0000a0/app-root/runtime/repo/node_modules/bower/lib/node_modules/mkdirp/index.js:76:24)
remote:     at Object.create.all.get (/var/lib/openshift/56df98da7628e17b0c0000a0/app-root/runtime/repo/node_modules/bower/lib/node_modules/configstore/index.js:41:13)
remote:     at Object.Configstore (/var/lib/openshift/56df98da7628e17b0c0000a0/app-root/runtime/repo/node_modules/bower/lib/node_modules/configstore/index.js:27:44)
remote:     at readCachedConfig (/var/lib/openshift/56df98da7628e17b0c0000a0/app-root/runtime/repo/node_modules/bower/lib/config.js:19:23)
remote:     at defaultConfig (/var/lib/openshift/56df98da7628e17b0c0000a0/app-root/runtime/repo/node_modules/bower/lib/config.js:11:12)
remote:     at Object.<anonymous> (/var/lib/openshift/56df98da7628e17b0c0000a0/app-root/runtime/repo/node_modules/bower/lib/index.js:16:32)
remote:     at Module._compile (module.js:413:34)
remote:
```

**Comment by notthetup** [comment]
http://christian.sterzl.info/blog/bower-on-openshift

**Comment by sayanee** [comment]
closed with #240
